src/main/java/io/vertx/json/schema/impl/SchemaRepositoryImpl.java

```diff
@@ -88,6 +118,34 @@ public SchemaRepository dereference(String uri, JsonSchema schema) throws Schema
     return this;
   }
 
+  @Override public SchemaRepository preloadMetaSchema(FileSystem fs, Draft draft) {
+    List<String> metaSchemaIds;
```

Comment:
Should we have an overloading here where we can get the draft from the options?

Say a second method:

```java
public SchemaRepository preloadMetaSchema(FileSystem fs) {
  return preloadMetaSchema(fs, options.getDraft());
}
```

Reply:
Good idea, but we should throw an exception in case no draft is set in the options.

Should it be an IllegalArgumentException or IllegalStateException? Because the argument we would "miss" is not passed, so I guess IllegalStateException is more fitting.

Reply:
Yes, `IllegalState` sounds reasonable to me. I'd use `IAE` if we're validating the method arguments

Reply:
Hi, I added the proposed method. Please have a look, if this meets your expectations.